certifier_service/certprotos/certifier.proto

```diff
@@ -247,15 +253,20 @@ message proof {
   repeated proof_step steps                 = 3;
 };
 
+message accelerator_group {
+  optional string type                      = 1;
```

Comment:
we may need to include other fields like uuid, driver version, vbios version, etc.